src/dragonball/src/dbs_pci/src/device.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023 Alibaba Cloud. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::any::Any;
+
+#[cfg(target_arch = "aarch64")]
+use dbs_device::resources::DeviceResources;
+use dbs_device::DeviceIo;
+use downcast_rs::Downcast;
```

Comment:
Do we really need this?

Reply:
nope, but this will be used in later PRs and it's basic functionality.